### PR TITLE
chore: Bump sam translator version, boto3 version bump to 1.15

### PR DESCRIPTION
**Pull request by qingchm**
#### Why is this change necessary?
Required in SAM release 1.30.0

#### Checklist

- [ ] Write design document ([Do I need to write a design document?](https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#design-document))
- [ ] Write unit tests
- [ ] Write/update functional tests
- [ ] Write/update integration tests
- [x] `make pr` passes
- [x] `make update-reproducible-reqs` if dependencies were changed
- [ ] Write documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).

### CODE REVIEW

**Review by aahung — APPROVED** [review]

File: requirements/base.txt [hunk 1]
```diff
@@ -2,11 +2,11 @@ chevron~=0.12
 click~=7.1
 Flask~=1.1.2
 #Need to add Schemas latest SDK.
-boto3~=1.14.0, >=1.14.23
+boto3~=1.15.0
```
Comment (aahung) [inline requirements/base.txt:5]: Since we found why there was conflict before, we can revert this one

Comment (aahung) [reply requirements/base.txt:5]: or bump it to 1.16

**Review by c2tarun — APPROVED** [review]

File: requirements/base.txt (hunk 1 above)
Comment (IrinaAlexandra) [reply requirements/base.txt]: What happened here? Can it be bumped to 1.16 now?

Comment (aahung) [reply requirements/base.txt]: it had conflict with samtranslator's boto3 requirements. It was fixed.


> Can it be bumped to 1.16 now?

yes we will. Is there any reason you want us to bump this version?

Comment (IrinaAlexandra) [reply requirements/base.txt]: I understand. The reason I am asking is that by the time this change was made, boto3 was at 1.16.x. This in turn puts us in a position where we need to either use an older version of `aws-sam-cli` (which is not ideal) or an older version of `boto3` (also not ideal).

Comment (aahung) [reply requirements/base.txt]: Oh I see. Are you using pip to install sam-cli? We recommend to use homebrew (macOS, linux) or installer (windows) to install it. In that case, you would not have this dependencies issue.

Comment (aahung) [reply requirements/base.txt]: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html

Comment (aahung) [reply requirements/base.txt]: Besides, I created a PR to loose the boto3's version https://github.com/aws/aws-sam-cli/pull/2493

**Pull request merged**